cli/src/pcluster/api/util.py

```diff
@@ -154,13 +157,50 @@ def _deserialize_dict(data, boxed_type):
     return {k: _deserialize(v, boxed_type) for k, v in six.iteritems(data)}
 
 
-def assert_node_executable():
+def assert_valid_node_js():
+    _assert_node_executable()
+    _assert_node_version()
+
+
+def _assert_node_executable():
     node_exe = shutil.which("node")
-    LOGGER.debug("Found nodejs executable in %s", node_exe)
+    LOGGER.debug("Found Node.js executable in %s", node_exe)
     if not node_exe:
         message = (
             "Unable to find node executable. Node.js is required by the AWS CDK library used by ParallelCluster, "
             "see installation instructions here: https://docs.aws.amazon.com/parallelcluster/latest/ug/install-v3.html"
         )
         LOGGER.critical(message)
         raise Exception(message)
+
+
+def _assert_node_version():
+    try:
+        node_version = subprocess.check_output(  # nosec
```

Comment:
why we need `nosec`?

Reply:
added a comment, can you please double check?